extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -497,6 +494,46 @@ export class AmpStoryPage extends AMP.BaseElement {
     ]);
   }
 
+  /** @override */
+  onMeasureChanged() {
```

Comment:
Do you know to what extent calls to this method are throttled/debounced?

Reply:
I couldn't find an obvious answer reading the code, maybe @jridgewell would know?

Reply:
They're not. It's called every time `Resources` measures it, and the measurements are different than the last time.